cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/listener/TaskConfigListener.java

```diff
@@ -0,0 +1,15 @@
+package de.dytanic.cloudnet.ext.bridge.listener;
+
+import de.dytanic.cloudnet.driver.event.EventListener;
+import de.dytanic.cloudnet.event.service.task.ServiceTaskAddEvent;
+
+public class TaskConfigListener {
+
+    @EventListener
+    public void handleTaskAdd(ServiceTaskAddEvent event) {
+        if(!event.getTask().getProperties().contains("requiredPermission")) {
+            event.getTask().getProperties().append("requiredPermission", "null");
```

Comment:
Please actually set null, not just a string containing null.

Reply:
same from the one above, didnt knew how to just "null" is not a option there, tried setting a empty one, didnt work.

Reply:
We could add something like appendNull, but right now you can just append null casted to a String.

Reply:
I've added a very dirty version of "appendNull" which actually casts a null to a string

Reply:
Just add JsonNull.INSTANCE to the jsonObject

Reply:
Yup. Done in latest push